.github/ISSUE_TEMPLATE/courses.yaml

```diff
@@ -0,0 +1,83 @@
+name: \[COURSES\] Typo report or new content request
+description: Report typos, bugs, out-of-date content, broken links, etc. or make a request for new content
+labels: ["content 📄", "course 🎓", "needs triage 🤔"]
+assignees:
```

Comment:
should we assign Chris Porter as well?

Reply:
He doesn't have a public handle yet!! Spoke to him about it yesterday, and he will create one